### ISSUE TITLE
How to resize two images one wrt  left 0,0 and  other wrt right top

### ISSUE THREAD

**Issue by vkkindia** [jcupitt/libvips#1036]
I need to resize two separate images 
one wrt  left 0,0  or left centre or left bottom
and other wrt right top or centre or bottom
The objective is to place two resized images side by side but not in a single file.

**Comment by jcupitt** [comment]
I don't think anyone does anything but top-left for corner convention. For the other corners, you can just displace the output by one pixel in whatever direction.

Are you certain you mean corner convention? 

Consider this 3x3 pixel image:

![x](https://user-images.githubusercontent.com/580843/42809372-6f1c0436-89ad-11e8-897e-492731fddf9b.png)

If I upsize with centre convention I get:

![x](https://user-images.githubusercontent.com/580843/42809464-a03335d0-89ad-11e8-894e-4940fab8a0ec.png)

```
$ vipsthumbnail 3x3.v --size 96 -o x.png
```

If I upsize with top-left corner convention I get:

![x](https://user-images.githubusercontent.com/580843/42809637-f96a721c-89ad-11e8-9b55-684e978901ce.png)

```
vips affine 3x3.v x.png "32 0 0 32" --interpolate bilinear
```

Is that what you mean?

**Comment by vkkindia** [comment]
Yes ofcourse, the idea is to place two resized images side by side, the two images will be closer.

**Comment by jcupitt** [comment]
Well, it seems odd to me, but try:

```
vips affine 3x3.v x.png "32 0 0 32" --interpolate bicubic --oarea "0 0 96 96" --idx 1 --idy 1
```

To make:

![x](https://user-images.githubusercontent.com/580843/42812727-87deb50a-89b6-11e8-8246-d6a6ea081114.png)

The `--oarea` sets the area of output pixels that will be written. Without this, `affine` will output the bounding box of the transformation of the input area. The `--idx`, `--idy` set an input displacement --- in this case, (1, 1) will select bottom-right convention.

**Comment by vkkindia** [comment]
Thanks Cupit, Iam still leaning....
One more doubt, if I use --idx 0.5 and --idy 0.5 will it become center
convention?

**Comment by jcupitt** [comment]
Yes, that's right.
